command/agentproxyshared/cache/lease_cache.go

```diff
@@ -337,6 +416,15 @@ func (c *LeaseCache) Send(ctx context.Context, req *SendRequest) (*SendResponse,
 		return resp, nil
 	}
 
+	// TODO: if secret.MountType == "kvv1" || secret.MountType == "kvv2"
```

Comment:
See https://github.com/hashicorp/vault/pull/23047

Reply:
In addition to that condition, remember that we'll have to think more about method type: DELETE, PATCH, etc.